_posts/2020-02-06-how-to-configure-a-production-grade-ci-cd-setup-for-apps-and-infrastructure-code.adoc

```diff
@@ -0,0 +1,2045 @@
+---
+title: How to configure a production-grade CI/CD workflows for application and infrastructure code
+categories: Foundations
```

Comment:
Perhaps "Automations and Workflows" instead? As per use case pages: https://github.com/gruntwork-io/team-product/blob/master/rfcs/initial-use-case-pages.md#automation-and-workflows